google/cloud/internal/populate_common_options_test.cc

```diff
@@ -127,6 +128,37 @@ TEST(PopulateCommonOptions, EndpointAuthority) {
   }
 }
 
+TEST(PopulateCommonOptions, UniverseDomain) {
+  auto actual =
+      PopulateCommonOptions(Options{}.set<UniverseDomainOption>("my-ud.net"),
+                            {}, {}, {}, "default.googleapis.com");
+  EXPECT_EQ(actual.get<EndpointOption>(), "default.my-ud.net");
```

Comment:
Pondering: should the output be `default.my-ud.net.` (with a trailing `.`)?

Reply:
I think what you are asking is more general: Should the endpoint always end in `.`?

I don't know the answer. I am no expert on FQDNs.

Note that we use the application-supplied value in the `EndpointOption` as is, so doing the same with `UniverseDomainOption` makes sense to me.